herbert_core/classes/MPIFramework/@ClusterWrapper/ClusterWrapper.m

```diff
@@ -466,7 +466,8 @@
                     paused  = false;
                     running = false;
                     failedC = true;
-                    messC   = FailedMessage('Job Initialization process have failed or has not been started');
+                    messC   = FailedMessage( ...
```

Comment:
it will be good NOT to fix formatting like this - it adds noise to the review process and slows down the fixing process

Reply:
The development is not for reviewer convenience. Proper